__tests__/atlas.test.ts

```diff
@@ -147,10 +147,6 @@ describe('run with "latest" flag', () => {
         Dir: res.summary?.Env.Dir
       },
       Steps: [
-        {
-          Name: 'Migration Integrity Check',
-          Text: 'File atlas.sum is valid'
-        },
```

Comment:
not related to this PR,
failed the tests on master branch, so I fixed it on the way

Reply:
Can you explain what changed?